### PR TITLE
Countdown to the New Year: 31 Days of Ember Addons - ember-page-title (Dec 11th 2019)

### PR DESCRIPTION
**Pull request by venusang**

### CODE REVIEW

File: source/2019-12-11-countdown-to-the-new-year-ember-page-title.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+**This is the eleventh in our DecEmber series- "Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
+
+## Day 11
+
+This addon [ember-page-title](https://emberobserver.com/addons/ember-page-title) provides a helper for changing the browser tab title of the page you're on within your Ember single-page application (SAP).
+
+### What It Does
+
+The `ember-page-title` addon provides a zero-config way to use npm packages in your Ember app, and even lazy-load them.
```
Comment (MelSumner) [inline source/2019-12-11-countdown-to-the-new-year-ember-page-title.md]:
```suggestion
The `ember-page-title` addon provides a zero-config way to use npm packages in your Ember app, and even lazy-load them. This is helpful for many reasons, but an important reason is that it is a [WCAG success criteria (2.4.2)](https://www.w3.org/WAI/WCAG21/Understanding/page-titled) to provide a page title. 

A quick tip about page titles! The title of each Web page should:

- Identify the subject of the Web page
- Make sense when read out of context, for example by a screen reader or in a site map or list of search results
- Be short

```

**Review by MelSumner — CHANGES_REQUESTED** [review]

File: source/2019-12-11-countdown-to-the-new-year-ember-page-title.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Identify the subject of the Web page
+- Make sense when read out of context, for example by a screen reader or in a site map or list of search results
+- Be short
+
+### Why I Like It
+
+Thanks to `ember-page-title`, you can easily provide users with a simple HTML browser title tag so they know where they are within your single-page application (SPA).
+
+All you need to do is run <code>`ember install ember-page-title`</code>, add <code>{{head-layout}}</code> to your <code>application.hbs</code> file and finally include the title helper to each page section template, for example if you had an About section, <code>{{title "About"}}</code>. This helper also gives you the ability to add custom separators, for example <code>{{title "About" separator=">"}}</code>. More information can be found at the [Ember Page Title Docs Site](https://adopted-ember-addons.github.io/ember-page-title/).
```
Comment (MelSumner) [inline source/2019-12-11-countdown-to-the-new-year-ember-page-title.md]:
```suggestion
How to use it: 

1. Run `ember install ember-page-title`
2. Add `{{head-layout}}` to your `application.hbs` file 
3. Include the title helper to each page's template. For example, if you had an "About" route, add `{{title "About Us"}}` to that template. This helper *also* gives you the ability to add custom separators, for example `{{title "About" separator=">"}}`. 

More information can be found at the [Ember Page Title Docs Site](https://adopted-ember-addons.github.io/ember-page-title/).
```

File: source/2019-12-11-countdown-to-the-new-year-ember-page-title.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+This addon is Octane-ready! <check to make sure it is>
```
Comment (MelSumner) [inline source/2019-12-11-countdown-to-the-new-year-ember-page-title.md]: @kategengler do you know if this addon is Octane ready?

**Review by MelSumner — CHANGES_REQUESTED** [review]
Looking Good! :+1:

File: source/2019-12-11-countdown-to-the-new-year-ember-page-title.md (hunk 2 above)
Comment (venusang) [reply source/2019-12-11-countdown-to-the-new-year-ember-page-title.md]: @MelSumner  - this is a lot easier to read- thank you for the suggestion.

File: source/2019-12-11-countdown-to-the-new-year-ember-page-title.md (hunk 3 above)
Comment (venusang) [reply source/2019-12-11-countdown-to-the-new-year-ember-page-title.md]: @MelSumner fwiw, I tried this out in an Octane app that I spun up and it was working for me.  I also tried out the {{head-layout}} in Angle Bracket notation.  But I'm not sure if this is sufficient to confirm on my end that it's Octane ready.

**Review by MelSumner — APPROVED** [review]
Thank you for writing this! :+1:

**Pull request merged**